views/pages/urls/index.hbs

```diff
@@ -1,5 +1,19 @@
 {{!< center-container }}
 
+<nav aria-label="Page navigation example">
+    <ul class="pagination justify-content-end">
+        <li class="page-item {{#unless pagination.hasPrev}}disabled{{/unless}}">
+            <a class="page-link" href="?page={{ dec pagination.page }}" tabindex="-1">Previous</a>
+        </li>
+        {{#range pagination.pageCount}}
```

Comment:
isn't this something we can handle with an each loop ? handlebar already does have looping support

Reply:
Actually rather than iterating over an iterable I wanted a counter loop! Couldn't find the work around with `#each`